pkg/cmd/server/kubernetes/node/node.go

```diff
@@ -82,73 +82,67 @@ func (c *NodeConfig) EnsureDocker(docker *dockerutil.Helper) {
 	if c.KubeletServer.ContainerRuntime != "docker" {
 		return
 	}
-	dockerClient, dockerAddr, err := docker.GetKubeClient(c.KubeletServer.RuntimeRequestTimeout.Duration, c.KubeletServer.ImagePullProgressDeadline.Duration)
-	if err != nil {
-		c.HandleDockerError(fmt.Sprintf("Unable to create a Docker client for %s - Docker must be installed and running to start containers.\n%v", dockerAddr, err))
-		return
+
+	var endpoint string
+	if len(os.Getenv("DOCKER_HOST")) > 0 {
```

Comment:
Seems better suited like a config variable rather than an env.

Reply:
> Seems better suited like a config variable rather than an env.

Pre-existing as upstream helpers were removed.  This can't change without breaking previously working environments.